primitives/src/lib.rs

```diff
@@ -8,8 +8,6 @@
 //!
 //! [`rust-bitcoin`]: <https://github.com/rust-bitcoin>
 
-// NB: This crate is empty if `alloc` is not enabled.
```

Comment:
Why not keep it? IMO it makes the commit more confusing and no-alloc support could be done properly in a separate PR.

Reply:
I guess it could've been done in a separate commit, but it makes sense to say "now that we are importing functionality that does not require alloc, we might as well enable no-alloc".

As far as "doing it properly", I think that just means strictly increasing the set of no-alloc functionality, which can be done later.

Reply:
My point is that if a commit claims to only move stuff around I don't expect it to mess with feature gates. Whatever, I've ACKed, just providing a different point of view.

Reply:
Thanks, if it made the PR in anyway harder to review it shouldn't have been in there. These big moves are hard enough as it is. Thanks for the acks.